edgexfoundry/edgex-compose
# feat: Upgrade to eKuiper 1.9

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
Using this branch run `make run no-secty ds-virtual mqtt-broker`
Run mqtt.fx connected to the MQTT Broker and subscribed to `results` topic
Send the following to ekuiper:
- http://localhost:59720/streams
```
{
  "sql": "create stream EdgexStream () WITH (FORMAT=\"JSON\", TYPE=\"edgex\")"
}
```
- http://localhost:59720/rules
```
{
  "id": "anyrule",
  "sql": "SELECT * FROM EdgexStream",
  "actions": [
    {
      "mqtt": {
        "server": "tcp://edgex-mqtt-broker:1883",
        "topic": "result",
        "clientId": "demo_001"
      }
    }
  ]
}
```
Verify the eKuiper data is received in mqtt.FX
Sample: `[{"Bool":false}]`
